java_client/ChitChatDesktop/src/com/moc/chitchat/view/main/SearchPane.java

```diff
@@ -0,0 +1,162 @@
+package com.moc.chitchat.view.main;
+
+import com.mashape.unirest.http.exceptions.UnirestException;
+import com.moc.chitchat.controller.authentication.UserSearchController;
+import com.moc.chitchat.exception.UnexpectedResponseException;
+import com.moc.chitchat.model.UserModel;
+import com.moc.chitchat.view.authentication.AuthenticationStage;
+import com.moc.chitchat.view.authentication.BaseView;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.control.*;
+import javafx.scene.input.MouseEvent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import org.tbee.javafx.scene.layout.fxml.MigPane;
+
+import javax.xml.soap.Text;
+import java.util.List;
+import java.util.Optional;
+
+
+/**
+ *
+ */
+@Component
+public class SearchPane extends BaseView implements EventHandler<ActionEvent> {
+
+    private UserSearchController userSearchController;
+
+    private TextField usernameField,messagesend;
+    private Button searchBtn,sendBtn, logoutBtn;
+    private ListView<String> searchlist;
+    private ObservableList<String> names;
+    private TextArea DisplayMessages;
+    private MainStage stage;
+    private AuthenticationStage stag;
+    private Text chatTitle;
+
+
+    @Autowired
+    public SearchPane(
+            UserSearchController userSearchController
+    ) {
+        this.userSearchController = userSearchController;
+    }
+
+
+    public void setStage(MainStage stage) {
+               this.stage = stage;
+    }
+
+
+
+    public MigPane getContentPane() {
+        MigPane searchPane = new MigPane();
+
+        MigPane searchForm = new MigPane();
+
+
+        ///Search AREA @@@@@@@@@@@???????????????????/////
+        this.usernameField = new TextField();
+        this.usernameField.setPromptText("Find User");
+        searchForm.add(this.usernameField, "span, cell 0 2 , align left");
+
+        this.searchBtn = new Button("Search");
+        this.searchBtn.setOnAction(this);
+        searchForm.add(this.searchBtn, "cell 0 3, grow");
+
+
+
+        this.searchlist = new ListView<String>();
+        searchForm.add(this.searchlist,"grow,cell 0 1");
+        this.searchlist.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                System.out.println("you clicked the "+searchlist.getSelectionModel().getSelectedItem());
+
+            }
+        });
+
+        //Chat- AREA""""""""
```

Comment:
The Chat area etc. should not be here. This **Pane** is purely for searching. Could you remove this and we'll add it to a `ChatPane` class in the `feature/user-sends-message` branch. Same with the Menu Bar.